library/std/src/sys/windows/c.rs

```diff
@@ -1020,77 +1020,87 @@ extern "system" {
     pub fn HeapAlloc(hHeap: HANDLE, dwFlags: DWORD, dwBytes: SIZE_T) -> LPVOID;
     pub fn HeapReAlloc(hHeap: HANDLE, dwFlags: DWORD, lpMem: LPVOID, dwBytes: SIZE_T) -> LPVOID;
     pub fn HeapFree(hHeap: HANDLE, dwFlags: DWORD, lpMem: LPVOID) -> BOOL;
+
+    // >= Vista / Server 2008
+    // https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-createsymboliclinka
+    pub fn CreateSymbolicLinkW(
+        _lpSymlinkFileName: LPCWSTR,
+        _lpTargetFileName: LPCWSTR,
+        _dwFlags: DWORD,
```

Comment:
Not that it matters much, but the underscores were probbaly originally added because these arguments were not used in the `compat_fn` implementations, to avoid warnings about them being unused. They're no longer necessary. (Same for the next three functions.)